scripts/docker-dos/dosUrlLocalizer.sc

```diff
@@ -78,16 +82,26 @@ def resolveDosThroughMartha(dosUrl: String, marthaUrl: Uri) : Try[MarthaResponse
   import MarthaResponseJsonSupport._
 
   val requestBody = json"""{"url":$dosUrl}"""
+  val scopes = Lists.newArrayList("https://www.googleapis.com/auth/userinfo.email",
+    "https://www.googleapis.com/auth/userinfo.profile")
 
   val credentials = GoogleCredentials.getApplicationDefault()
-  val accessToken = credentials.refreshAccessToken().getTokenValue()
+  val scopedCredentials = credentials.createScoped(scopes)
+  val accessToken = scopedCredentials.refreshAccessToken().getTokenValue()
+
+  val longTimeoutConfig =
+    BlazeClientConfig
+      .defaultConfig
+      .copy(idleTimeout = 5.minutes,
+        responseHeaderTimeout = 5.minutes,
+        requestTimeout = 5.minutes)
```

Comment:
TOL except for `idleTimeout` aren't those values kind of high?

Reply:
I could be wrong, but I thought I heard that Martha has to talk to several services and may end up taking > 60s to complete(? or maybe start?) an http response.